Makefile

```diff
@@ -116,7 +116,7 @@ update-linters:
 
 update-mockgen:
 	@printf $(COLOR) "Install/update mockgen tool..."
-	@go install github.com/golang/mock/mockgen@v1.6.0
+	@go install github.com/golang/mock/mockgen@v1.7.0-rc.1
```

Comment:
any feature we need in this rc version?

Reply:
the ability for mockgen to handle generics: https://github.com/temporalio/temporal/pull/4097/files#diff-1f9d74f27f59376896675188a3673cac1730b7afc79e890ca49f0da83b3f23e6R105